src/app/profile/overview/spaces/spaces.component.ts

```diff
@@ -19,7 +19,7 @@ export class SpacesComponent implements OnDestroy, OnInit  {
   loggedInUser: User;
   pageSize: number = 20;
   subscriptions: Subscription[] = [];
-  spaceToDelete: Space;
+  // spaceToDelete: Space;
```

Comment:
Instead of a comment, we should just remove this line.

Reply:
I left this line so that it matches with the delete space code that was also commented out.